tests/js/app/modules/layout/testArticleStack.js

```diff
@@ -31,6 +31,9 @@ describe('Layout.ArticleStack', function () {
         let factory;
         [module, factory] = new MockFactory.setup_tree({
             type: ArticleStack.ArticleStack,
+            references: {
+                'selection': 'selection',
+            },
```

Comment:
...and it would be good to add a test that ensures the module still works without the reference.